database/index.js

```diff
@@ -14,7 +14,16 @@ const photoSchema = mongoose.Schema({
 });
 const Photo = mongoose.model('Photo', photoSchema);
 
+const getAllPhotos = async () => await Photo.find({}).exec();
+
+const getPhotoById = async id => await Photo.findOne({ id });
+
+const getPhotosByWorkspaceId = async workspaceId => await Photo.find({ workspaceId }).exec();
```

Comment:
doesn't effect functionality but mongoose queries are awaitables/thenables so you can just return them here without using await or exec and handle the async part wherever you implement them instead.